Source/UI/NewUI/CommonUI/View/Component/InputView.swift

```diff
@@ -6,34 +6,43 @@ class InputView: UIView {
 
     weak var delegate: TextFieldViewDelegate?
     private(set) var style: CellTextFieldStyle?
-    private(set) lazy var textFieldContainerBottomAnchor = textFieldContainer?.bottomAnchor.constraint(equalTo: bottomAnchor, constant: 0)
+    private(set) lazy var textFieldContainerBottomAnchor = textFieldContainer.bottomAnchor.constraint(equalTo: bottomAnchor, constant: 0)
 
     // MARK: - UI elements
 
-    private(set) lazy var headerLabel: LabelView? = {
+    private(set) lazy var headerLabel: LabelView = {
         LabelView().disabledAutoresizingIntoConstraints()
     }()
 
-    private(set) lazy var mandatoryLabel: LabelView? = {
+    private(set) lazy var mandatoryLabel: LabelView = {
         LabelView().disabledAutoresizingIntoConstraints()
     }()
 
-    private(set) lazy var hintLabel: LabelView? = {
+    private(set) lazy var hintLabel: LabelView = {
         LabelView().disabledAutoresizingIntoConstraints()
     }()
 
-    private(set) lazy var textFieldContainer: UIView? = {
-        let view = UIView().disabledAutoresizingIntoConstraints()
+    private(set) lazy var textFieldContainer: UIStackView = {
+        let view = UIStackView().disabledAutoresizingIntoConstraints()
+        view.isLayoutMarginsRelativeArrangement = true
+        view.layoutMargins = UIEdgeInsets(top: 0, left: 20, bottom: 0, right: 20)
+        view.spacing = 16
```

Comment:
In case Apple decides that all stack views should default to vertical? That would be a Y2K level disaster. Planes falling out of the sky etc.

I'll add it but the default can never change at this point.

Reply:
> That would be a Y2K level disaster. Planes falling out of the sky etc.

Not sure iOS will be used on airplanes or to sync the world clock, but I see where you're going with it. 

The reason I brought it up is that Swift code and frameworks can now run on multiple OSs and hardware configurations (phones, watches, tablets, desktops - each with their own OS versioning history and roadmap). An example that comes to my mind is SwiftUI, where the modifier (I may be mixing them) `listStyle` makes a list appear very different between a phone and a tablet. Then you have very different behaviour for tab bars and menus, especially when you throw in localisations, screens, hardwares, etc.

Sure, we are pretty confident based on what we know now that the axis default will not be impacted on any known configurations at this moment in time. But with our code starting to scale beyond just iPhones and the ecosystem evolving, all I'm suggesting is having discussions and considering these aspects, which is why I did not ask for the change, it was just a suggestion. Feel free to disregard the specific case here, but try to always have at the back of the mind the bigger picture.